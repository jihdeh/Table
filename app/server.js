const express = require('express');

const app = express();

app.set('port', (process.env.PORT || 3001));

app.use(express.static('client/build'));

const server = app.listen(app.get('port'));

module.exports = {
  server: server
};
