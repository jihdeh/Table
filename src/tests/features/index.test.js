import app from '../../server';

it('app is the same', () => expect(JSON.stringify(app)).toMatchSnapshot());
 app.close();
