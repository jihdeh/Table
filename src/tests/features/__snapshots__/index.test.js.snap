exports[`test app is the same 1`] = `"{\"domain\":null,\"_events\":{},\"_eventsCount\":3,\"_connections\":0,\"_handle\":null,\"_usingSlaves\":false,\"_slaves\":[],\"_unref\":false,\"allowHalfOpen\":true,\"pauseOnConnect\":false,\"httpAllowHalfOpen\":false,\"timeout\":120000,\"_pendingResponseData\":0,\"_connectionKey\":\"6::::3001\"}"`;
